sci-libs/blis/blis-1.0-r1.ebuild

```diff
@@ -0,0 +1,183 @@
+# Copyright 2019-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{11..13} )
+inherit python-any-r1 toolchain-funcs
+
+DESCRIPTION="BLAS-like Library Instantiation Software Framework"
+HOMEPAGE="https://github.com/flame/blis"
+SRC_URI="https://github.com/flame/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~ppc64 ~x86"
+CPU_USE=(
+	cpu_flags_ppc_{vsx,vsx3}
+	cpu_flags_arm_{neon,v7,v8,sve}
+	cpu_flags_x86_{ssse3,avx,fma3,fma4,avx2,avx512vl}
+)
+IUSE="doc eselect-ldso index64 openmp pthread serial static-libs ${CPU_USE[@]}"
+REQUIRED_USE="?? ( openmp pthread serial )"
+
+DEPEND="
+	eselect-ldso? (
```

Comment:
I think eventually this can become just an IDEPEND, but I'm not sure how well blockers will work there, so we'll do that in a while.

Reply:
To be honest, I didn't want to spend more effort than necessary on `eselect-ldso` stuff, because presumably it's going away anyway.